docs/architecture/parts.md

```diff
@@ -15,26 +15,43 @@ aliases:
 When {{< glossary_tooltip term_id="machine" text="smart machines" >}} communicate with each other, they can share resources and operate collaboratively.
 This document explains how to establish secure connections between machines.
 
-### Machine parts
+## Machine parts
 
-Machines are organized into _parts_, where each part represents a computer (a single-board computer, desktop, laptop, or other computer) running [`viam-server`](/installation/), the hardware [components](/components/) attached to it, and any [services](/services/) or other resources running on it.
+Machines are organized into _parts_, where each part represents a computer (a single-board computer, desktop, laptop, or other computer) running [`viam-server`](/installation/), the hardware [components](/components/) attached to it, and any [services](/services/) or other {{< glossary_tooltip term_id="resource" text="resources" >}} running on it.
 
 Every smart machine has a main part which is automatically created when you create the machine.
 Multi-part machines also have one or more _sub-parts_ representing additional computers running `viam-server`.
 
-There are two ways to link machine parts:
+There are two ways to link machine parts, depending on whether you want to think of your parts as all part of one machine, or be able to share resources across multiple machines:
```

Comment:
I think this change makes it more confusing to me. I think either stick with how it was and explain the rest as it is in the bullets or maybe something like this?

```suggestion
You can think of parts as machine parts that together make up a whole machine where each part can also be shared as a resource across multiple machines:
```

Reply:
What I was trying to get across is that you could just use remotes for all cases, but making a multi-part machine is just kind of an organizational concept.

Reply:
```suggestion
There are two ways to link machine parts:
```